site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -223,12 +223,18 @@ The issuer used by service providers (i.e. Google, Zendesk, etc.) to identify th
 [field]#Audience# [optional]#Optional#::
 Some service providers require a different audience (such as Zendesk). You can leave this blank if the audience is the same as the issuer.
 
-[field]#Callback URL (ACS)# [required]#Required#::
+[field]#Authorized redirect URLs (ACS)# [required]#Required#::
 The URL that FusionAuth returns the user to after they have logged in via SAML v2, also known as the Assertion Consumer Service URL (ACS).
 
 [field]#Logout URL# [optional]#Optional#::
 The URL that the user is redirected to after they are logged out. Usually this is the starting location of the application.
 
+[field]#Debug enabled# [optional]#Optional#::
+Enable debug to create an event log to assist you in debugging integration errors.
+
+[field]#Require signature# [optional]#Optional#::
+Enable to required the SAML v2 Service Provider to sign the SAML v2 authentication request. When enabled, if a signature is omitted, the request will be rejected.
+
```

Comment:
Missing "Default verification key" field.

Reply:
Let me know if you approve of the way I structured the description for this.